lib/feature/home/create_task/bloc/create_task_bloc.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:bloc/bloc.dart';
+import 'package:flutter_template/data/repository/tasks/tasks_repository.dart';
+import 'package:flutter_template/log/log.dart';
+
+import 'create_task_event.dart';
+import 'create_task_state.dart';
+
+class CreateTaskBloc extends Bloc<CreateTaskEvent, CreateTaskState> {
```

Comment:
This seems like a good choice for Cubit - only one very simple event

Reply:
Just mind saving the user input in case the widget is rebuilt, like theme switch

Reply:
Thanks :) I modified it to use a Cubit